logger/build.gradle.kts

```diff
@@ -43,7 +43,7 @@ kotlin {
     sourceSets {
         val commonMain by getting {
             dependencies {
-                implementation("co.touchlab:kermit:1.0.0")
+                api("co.touchlab:kermit:1.1.1")
```

Comment:
Why the change from `implementation` to `api`?

Reply:
I think it's related to having this lib automatically imported on the consumers projects. 

I'm wondering if this could cause a dependency conflict in the future, in case another lib would import this `kermit` lib transitively 🤔

Reply:
because we are using it in the reloaded as we need to build the log writer from the library , and in theory the other clients will do the same to create a custom log writer